bin/proxy.js

```diff
@@ -0,0 +1,45 @@
+require('babel-register');
+
+const http = require('http');
+
+const bunyan = require('bunyan');
+const config = require('config');
+const httpProxy = require('http-proxy');
+
+const log = bunyan.createLogger({
+  name: 'proxy',
+  app: config.get('appName'),
+  serializers: bunyan.stdSerializers,
+});
+
+const proxy = httpProxy.createProxyServer();
+const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
```

Comment:
Is `https` the problem with using `addons-dev.allizom.org`? If so, could this script die in an informative error if `apiHost` starts with `https://` ?